code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1341,9 +1341,8 @@
 	return ..()
 
 /datum/reagent/nitrium_low_metabolization/on_mob_life(mob/living/carbon/M)
-	if(M.getStaminaLoss() > 0)
-		M.adjustStaminaLoss(-2 * REM, FALSE)
-		M.adjustToxLoss(1.5 *REM, FALSE)
+	M.adjustOrganLoss(ORGAN_SLOT_LIVER, 1)
```

Comment:
This will always deal damage and thus also affect high metabolization, revert the ``getStaminaLoss`` check.

Reply:
updated changelog

Reply:
This review isn't about the changelong

Reply:
the cost of having an antistun prepared all the time 🤷